src/main/kotlin/com/devfloor/untitled/article/domain/Article.kt

```diff
@@ -48,10 +50,15 @@ class Article(
     @JoinColumn(name = "author_id")
     val author: User = author
 
-    fun sliceContentByLength(length: Long): String =
-        if (content.length > length)
-            content.substring(startIndex = 0, endIndex = (length + 1).toInt())
-        else content
+    @ManyToOne
+    @JoinColumn(name = "board_id")
+    val board: Board = board
+
+    fun sliceContentByLength(length: Long): String = if (content.length > length) {
+        content.substring(startIndex = 0, endIndex = (length + 1).toInt())
+    } else {
+        content
+    }
```

Comment:
여기 if랑 else가 인덴트가 안맞으니까 뭔가 `if .. else` 구조로 한눈에 인지하기가 어려운거같은데
밑에처럼 바꾸면 어떨까??
```suggestion
    fun sliceContentByLength(length: Long): String =
        if (content.length > length) {
            content.substring(startIndex = 0, endIndex = (length + 1).toInt())
        } else {
            content
        }
```

Reply:
아 이거 봤는데 google kotlin 컨벤션에서 이런 식으로 사용하더라고. 그래서 똑같은 방식으로 쓰면 좋겠다 해서 수정한거 였는데 아래 방법이 좀 더 나을라나?

Reply:
https://developer.android.com/kotlin/style-guide?hl=ko
여기 참고한거??? 흠 뭐가 좋을지 잘 몰게따 @JoonBro 준형님은 어떠세요??

Reply:
이건 근데 한 줄 개행하는게 나은거 같아서 밑에 방식으로 수정할게!!!